instance/models/instance.py

```diff
@@ -89,6 +92,18 @@ def save(self, *args, **kwargs):
             'instance_id': self.pk,
         })
 
+    @classmethod
+    def can_manage(cls, user):
+        """
+        Returns true if the user is allowed to manage all instances.
+
+        User must:
+        * be a staff member, or
+        * have the "manage_all" permission
+        """
+        permission = '{}.{}'.format(cls._meta.app_label, "manage_all")
+        return user.is_staff or user.has_perm(permission)
```

Comment:
@bradenmacdonald You are right, my mistake.  In that case, I don't care whether to include `is_staff` here.